hooks/nrpe_helpers.py

```diff
@@ -23,6 +24,8 @@
 DB_KEY_KNOWN_REBOOT_TIME = "known_reboot_time"
 db = unitdata.kv()
 
+TAILORING_CIS_FILE = Path("/etc/usg/default-tailoring.xml")
```

Comment:
This constant is also defined in files/plugins/cron_cis_audit.py ; can they be consolidated to a single file?

Reply:
I couldn't find a way. The nrpe_helpers and the cron_cis should be independent. I tried to import from the folder files, but the charm cannot find and enters in error state.

Reply:
Ah got it. :+1: